src/test/java/mate/academy/dao/RoleDaoTest.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.dao.impl.RoleDaoImpl;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoTest extends AbstractTest {
```

Comment:
```suggestion
package mate.academy.dao.impl;

import java.util.Optional;
import mate.academy.dao.impl.RoleDaoImpl;
import mate.academy.model.Role;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class RoleDaoImplTest extends AbstractTest {
```
Since you testing RoleDaoImpl. same for UserDaoTest